### PR TITLE
Feat baseline readme

### PR DESCRIPTION
**Pull request by CrystalAQuick**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,29 @@
 # StardewValleyPocketTracker
+## Purpose
+This is an android application that is being created to gain experience developing applications
+in android studio, using git, solidifying programming fundamentals, and designing an application
```
Comment (assassinsgreed) [inline README.md]: > developing applications in android studio
can be 
> developing android applications written in Java

File: README.md [hunk 2]
```diff
@@ -1 +1,29 @@
...
+This is a fan-made project.
+Stardew Valley Pocket tracker keeps track of calender events in Stardew Valley.
+The app also keeps track of items that the user has acquired.
+
+
+## Visuals
+<insert some images of blueprint? or mockup?>
+
+## Installation
+Android version 4.0.3 (Ice Cream Sandwich) or higher is required.
```
Comment (assassinsgreed) [inline README.md:17]: If this is not intended for the play store, add information about pulling the code, building it locally, enabling developer options on your device, and installing the APK file.

If it is intended for the play store, put a comment about downloading it via the playstore (with a placeholder link) here.

File: README.md [hunk 3]
```diff
@@ -1 +1,29 @@
...
+
+## Usage
+<insert visual example of apps functionality>
+
+## Authors and Acknowledgment
+Developers
+    CrystalAQuick
```
Comment (assassinsgreed) [inline README.md]: Use bullet list notation here (see markdown cheat sheet)

File: README.md [hunk 4]
```diff
@@ -1 +1,29 @@
...
+    assassinsgreed
+
+
+
+
```
Comment (assassinsgreed) [inline README.md]: Remove excess extra lines (leave one blank line for code cleanliness though)

File: README.md [hunk 5]
```diff
@@ -1 +1,29 @@
 # StardewValleyPocketTracker
+## Purpose
```
Comment (assassinsgreed) [inline README.md:3]: A horizontal rule between the top level heading and the Purpose heading would be nice

**Review by assassinsgreed — COMMENTED** [review]
Looks good!  Just a few small cleanup suggestions.

File: README.md (hunk 2 above)
Comment (CrystalAQuick) [reply README.md:17]: intended for app store, will include placeholder link

**Pull request merged**